### PR TITLE
remove the kwargs of elu,relu6,pow,stanh,hard_sigmoid and swish

### PR DESCRIPTION
**Pull request by wzzju**
Remove the kwargs of elu,relu6,pow,stanh,hard_sigmoid and swish.

### CODE REVIEW

File: paddle/fluid/API.spec [hunk 1]
```diff
@@ -379,7 +379,7 @@ paddle.fluid.LoDTensor.__init__ 1. __init__(self: paddle.fluid.core.LoDTensor, a
...
+paddle.fluid.LoDTensor.set 1. set(self: paddle.fluid.core.Tensor, arg0: numpy.ndarray[float32], arg1: paddle::platform::CPUPlace) -> None  2. set(self: paddle.fluid.core.Tensor, arg0: numpy.ndarray[int32], arg1: paddle::platform::CPUPlace) -> None  3. set(self: paddle.fluid.core.Tensor, arg0: numpy.ndarray[float64], arg1: paddle::platform::CPUPlace) -> None  4. set(self: paddle.fluid.core.Tensor, arg0: numpy.ndarray[int64], arg1: paddle::platform::CPUPlace) -> None  5. set(self: paddle.fluid.core.Tensor, arg0: numpy.ndarray[bool], arg1: paddle::platform::CPUPlace) -> None  6. set(self: paddle.fluid.core.Tensor, arg0: numpy.ndarray[uint16], arg1: paddle::platform::CPUPlace) -> None  7. set(self: paddle.fluid.core.Tensor, arg0: numpy.ndarray[uint8], arg1: paddle::platform::CPUPlace) -> None  8. set(self: paddle.fluid.core.Tensor, arg0: numpy.ndarray[int8], arg1: paddle::platform::CPUPlace) -> None
```
Comment (panyx0718) [inline paddle/fluid/API.spec]: why is this line changed?

**Review by panyx0718 — APPROVED** [review]

**Review by qingqing01 — COMMENTED** [review]
better to have unit tests in https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/test_layers.py

**Review by qingqing01 — APPROVED** [review]

**Comment by panyx0718** [comment]
You can follow up a PR to add some simple test to make sure APIs are added correctly
